analysis/tools/InteractiveParser/src/main/kotlin/de/maibornwolff/codecharta/tools/interactiveparser/InteractiveParser.kt

```diff
@@ -0,0 +1,5 @@
+package de.maibornwolff.codecharta.tools.interactiveparser
+
+interface InteractiveParser {
+    fun callInteractive()
```

Comment:
The name `callInteractive` can create a bit of confusion. It can seem as if it's an alternative to the method `call` that's needed for the CLI which includes the logic of the parsers. This method however has nothing to do with the logic. I suggest `mainWithInteractiveDialog` or `executeWithInteractiveDialog`